web/src/data/api.ts

```diff
@@ -39,6 +39,15 @@ export type entityType = 'biosample'
  */
 export type entitySchemaType = keyof typeof NmdcSchema.$defs;
 
+/**
+ * Define a TypeScript type alias that accounts for all possible JSON values.
+ *
+ * References:
+ * - https://www.json.org/json-en.html
+ * - https://devblogs.microsoft.com/typescript/announcing-typescript-3-7/#more-recursive-type-aliases
+ */
+export type JSONValue = string | number | boolean | null | { [k: string]: JSONValue } | Array<JSONValue>;
+
```

Comment:
Is this being used anywhere? If not should it be removed (at least for now?)

Reply:
It is not being used anywhere. I will remove it. I think it will be useful as a more accurate type for the annotations property, but I didn't implement it there yet because that would open a "can of worms" downstream (where some code is written under the assumption that the current type is comprehensive, which I don't think is consistent with reality). I'll remove it from this PR branch and file an issue about my type concerns.

Reply:
Fixed (removed) in eab88f26484907bf0c7ee94e8903381404a99c79